lamindb/_record.py

```diff
@@ -224,14 +229,25 @@ def __init__(record: Record, *args, **kwargs):
                 message = _format_django_validation_error(record, e)
                 raise FieldValidationError(message) from e
     elif len(args) != len(record._meta.concrete_fields):
-        raise ValueError("please provide keyword arguments, not plain arguments")
+        sys.exit(
```

Comment:
Our convention is `raise SystemExit` or ideally one of the errors that inherit from it.

My hope is still that we find another way of surpressing the traceback (I have a thread somewhere experimenting with different Exception classes) and then it'll be hard to look for all occurances of `sys.exit()` in the code base and replace it with that solution.